models/wrf/work/input.nml

```diff
@@ -48,8 +48,8 @@
    first_obs_seconds        = -1,
    last_obs_days            = -1,
    last_obs_seconds         = -1,
-   num_output_state_members = 0,
-   num_output_obs_members   = 32,
+   num_output_state_members = 3,
+   num_output_obs_members   = 3,
```

Comment:
I'm not sure of the pros and cons of 0,32 vs 3,3 so if 3,3 seems appropriate to you that works.